.github/workflows/moodle-ci.yml

```diff
@@ -0,0 +1,111 @@
+name: Moodle Plugin CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Run CI on push to only `master`. Otherwise, all jobs will run twice on PRs.

See: https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662